(DOCSP-32967): Move Edge Server content into new section (#581)
mongodb/docs-app-services: source/edge-server/connect.txt

   :depth: 2
   :class: singlecol

Once the Edge Server is :ref:`configured <edge-server-configure>` and 
:ref:`running <start-and-stop-edge-server>`, you can connect to it from 
a client application.

.. _edge-server-connect-from-client:

Connect to the Edge Server from a Client
----------------------------------------

To connect to the Edge Server from a client, your app must:

- Use HTTP over port 80.

> - Set the Sync URL to the public accessible DNS address you set in the 

Reviewer: ```suggestion
- Set the Sync URL to the public-accessible DNS address you set in the 
```

>   Edge Server :ref:`config's <edge-server-install-and-configure>` ``hostname`` 

Reviewer: possible to link directly to that Complete the Edge Server Configuration Details step?

Author: Alas, seems we can't link directly to a step.

  field.

.. tip::

   While your Edge Server deployment is in development, we recommend connecting
   with HTTP over port 80 for simplicity. Before moving to production, 
   you can coordinate with your Product or Account Representative to configure
   HTTPS with a self-signed certificate.

Example: SwiftUI Template App
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

For a quick proof of concept using a template app, these are the 
modifications you can make to the :github:`Swift template app 
<mongodb/template-app-swiftui-todo>` to connect to the Edge Server:

.. procedure::

   .. step:: Configure the Server Connection

      In the Swift template app's :file:`atlasConfig`, set the values to:

      .. list-table::
         :header-rows: 1

         * - Key
           - Value

         * - ``baseUrl``

>            - The public-addressable DNS of your Edge Server's host. 

Reviewer: ```suggestion
           - The public-accessible DNS of your Edge Server's host. 
```
?

             This is ``http://`` followed by the same value you set in your 
             Edge Server config's ``hostname`` field.
          
         * - ``appId``

>            - The App ID of the App Services App you created above.

Reviewer: ```suggestion
           - The App ID of the App Services App you [link]created.
```

   .. step:: Adjust the App Transport Security Settings

      For iOS, Apple disables insecure connections by default. You must add

>       an exception for your public-addressable DNS to allow the client app 

Reviewer: ```suggestion
      an exception for your public-accessible DNS to allow the client app 
```
?

      to connect to it.

      Select your App target, and go to the :guilabel:`Signing & Capabilities`
      tab. Press the :guilabel:`+ Capability` button, and search for 
      ``App Transport Security Settings``. Add this to your app.

>       Add an exception domain for your public-addressable DNS. This should

Reviewer: ```suggestion
      Add an exception domain for your public-accessible DNS. This should
```
?

      be just the string domain, similar to: ``ec2-13-58-70-88.us-east-2.compute.amazonaws.com``.

      This creates an :file:`Info.plist` file in your project. Go to this file

>       to make additional adjustments.

Reviewer: ```suggestion
      to make the following additional adjustments.
```

   .. step:: Adjust the Log Level (Optional)

      You can adjust the log level if you'd like to get additional details 
      about the status of communication with the Edge Server.

      In the :file:`App` file, you can add an ``.onAppear`` block to set 
      the log level. Add a log level line in this block to get additional 
      information:

      .. code-block:: swift
         :emphasize-lines: 10-12

         var body: some Scene {
            WindowGroup {
               ContentView(app: app)
                  .environmentObject(errorHandler)
                  .alert(Text("Error"), isPresented: .constant(errorHandler.error != nil)) {
                     Button("OK", role: .cancel) { errorHandler.error = nil }
                  } message: {
                     Text(errorHandler.error?.localizedDescription ?? "")
                  }
                  .onAppear {
                     Logger.shared.level = .trace
                  }
            }
         }

      For more information about setting the log level, refer to 
      :ref:`swift-logging`.

   .. step:: Run the App

      Now you can build and run the app. When you log in, the app connects 

>       to the Edge Server. When you create new Items, those sync with the 

Reviewer: ```suggestion
      to the Edge Server. When you create new Items, those sync first with the 
```

>       Edge Server, and from there to the Atlas App Services App.

Reviewer: ```suggestion
      Edge Server, and then sync from the Edge Server to the Atlas App Services App.
```

Author: I think I can be a bit more accurate so I'm rewording this section a bit.

   :depth: 2
   :class: singlecol

Once the Edge Server is :ref:`configured <edge-server-configure>` and 
:ref:`running <start-and-stop-edge-server>`, you can connect to it from 
a client application.

.. _edge-server-connect-from-client:

Connect to the Edge Server from a Client
----------------------------------------

To connect to the Edge Server from a client, your app must:

> - Use HTTP over port 80.

Reviewer 2: if tls is not enabled
